src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a && ((!c && !d && b) || (!b && !d && c) || (!b && !c && d)))
+                || (!a && ((b && c && !d) || (c && d && !b) || (b && d && !c)));
```

Comment:
try to simplify it with binary operators

Reply:
done